src/gql/queries/get-patch-task-durations.graphql

```diff
@@ -0,0 +1,35 @@
+query PatchTaskDurations(
+  $patchId: String!
+  $sorts: [SortOrder!]
+  $page: Int
+  $variant: String
+  $taskName: String
+  $limit: Int
+) {
+  patchTasks(
+    patchId: $patchId
+    limit: $limit
+    page: $page
+    sorts: $sorts
+    variant: $variant
+    taskName: $taskName
+  ) {
+    count
+    tasks {
+      id
+      execution
+      status
+      displayName
+      buildVariantDisplayName
+      timeTaken
+      executionTasksFull {
+        id
+        execution
+        displayName
+        status
+        buildVariantDisplayName
+        timeTaken
```

Comment:
It doesn't look like this value is being resolved for execution tasks.
<img width="1076" alt="image" src="https://user-images.githubusercontent.com/4605522/164536695-a6cf971b-4ea5-4ee3-832a-5a77c47902b4.png">